lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -9,140 +9,143 @@
+        <div class="clearfix">
+        <div class="image">
+
+        <img src="${course_image_url(course)}"
+            alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} ${_('Cover Image')}" />
 
-<%block name="custom_content">
-
-<div class="container">
-  % if shoppingcart_items:
-        <%block name="billing_details_highlight">
-         % if order.order_type == 'business':
-            <li class="billing">${_('Billing Details')}</li>
-         % else:
-            <li class="billing hidden">${_('Billing Details')}</li>
-         % endif
-        </%block>
-
-     % if is_course_enrollment_closed:
-      <p id="expiry-msg">${_('{course_names} has been removed because the enrollment period has closed.').format(course_names=appended_expired_course_names)}</p>
-     % endif:
- <%
-    discount_applied = False
-    order_type = 'personal'
- %>
-
-  <section class="wrapper confirm-enrollment shopping-cart cart-view">
-    % for item, course in shoppingcart_items:
-       % if loop.index > 0 :
-        <hr>
-      %endif
-      % if item.order.order_type == 'business':
-       <% order_type = 'business' %>
-      %endif
-     <div class="user-data">
-      <div class="clearfix">
-      <div class="image">
-
-           <img style="width: 100%; height: 100%;" src="${course_image_url(course)}"
-                alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} Cover Image" />
-
-      </div>
-      <div class="data-input">
-        <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
-        <h1>${ course.display_name }</h1><span class="pull-right">${course.start_datetime_text()} -  ${course.end_datetime_text()}</span>
-        <hr />
-        <div class="three-col">
-            <div class="col-1">
-                % if item.list_price != None:
-                <% discount_applied = True %>
-                <div class="price">${_('Price per student:')} <span class="line-through">  ${currency_symbol}${"{0:0.2f}".format(item.list_price)}</span></div>
-                <div class="price green">${_('Discount Applied:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)} </span></div>
-                % else:
-                <div class="price">${_('Price per student:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)}</span></div>
-                % endif
-            </div>
-            <div class="col-2">
-              <div class="numbers-row">
-                <label for="students">${_('Students:')}</label>
-                <div class="counter">
-                    <input maxlength="3" title="Input qty and press enter." max="999" type="text" name="students" value="${item.qty}" id="${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}">
+        </div>
+        <div class="data-input">
+          <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
```

Comment:
@marjev - I think @sarina means to please add the comment to the file so that it may be seen by the translators. 

Having said that, this is not code that you've changed (other than it's indentation); so I'd prefer not to add new work to this PR by fixing all the pre-existing problems. Indeed, IMO this is trickier to address than a simple comment - the layout here in funky, and we really want the translators to translate `Registration for: ${course.display_name}` as a unit, so fixing it will require more changes.